modules/synthetic-protocol/src/lib.rs

```diff
@@ -425,7 +425,7 @@ impl<T: Trait> Module<T> {
 		collateral: T::Balance,
 	) -> SynthesisResult<T> {
 		let ratio = T::LiquidityPools::get_additional_collateral_ratio(pool_id, currency_id)
-			.ok_or(Error::<T>::NoAdditionalCollateralRatio)?;
```

Comment:
if `get_additional_collateral_ratio` return `Permill`, it will not hold the error `NoMinAdditionalCollateralRatio`

Reply:
Yes that's the point. By doing this we eliminate an error path. So one less way to go wrong.